### ISSUE TITLE
Connect to Remote ElasticSearch

### ISSUE THREAD

**Issue by jimbolya0607** [TheHive-Project/TheHive#501]
So this is an experience thing. 
I can install the hive and connect to ES on localhost address with no issues. 

However, I'm trying to set up ES to have more than one application talk to the ES server.  The ES server is not the same server as the Hive server.  I'd like to connect the Hive to this remote ES server.  
I've tried several network configurations including setting this at the end of the elasticsearch.yml:

network.host: 0.0.0.0
script.inline: on
cluster.name: hive
thread_pool.index.queue_size: 100000
thread_pool.search.queue_size: 100000
thread_pool.bulk.queue_size: 100000

Any suggestions for connecting Hive to a remote ES server would be great. 

Jimbo

**Comment by 3c7** [comment]
Hey @jimbolya0607,

please check the following documentation:
- https://www.elastic.co/guide/en/elasticsearch/reference/current/network.host.html
- https://github.com/TheHive-Project/TheHiveDocs/blob/master/admin/configuration.md#1-database

**Comment by nadouani** [comment]
Has this been answered?

**Comment by jimbolya0607** [comment]
Yes and No.
If you need to close this that's fine.
I'll keep working on it =)


Jimbo

**Comment by nadouani** [comment]
No, just asking, but if the problem is solved, just make sure to close the issue :)

**Comment by jimbolya0607** [comment]
Will do - I'm hoping to move forward on it this week.

Maybe once I get this solved we can post it on GitHub for others?

**Comment by alonek1** [comment]
Hi, Could you connect to remote elasticsearch ?
